### ISSUE TITLE
preventDefaultOnContextMenu not working in 25.2, 25.3

### ISSUE THREAD

**Issue by gcode-atwork** [ag-grid/ag-grid#4839]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```
**Reproducible: Steps**
- Use AG Grid version 25.2 or 25.3
- Happened to be using Angular, but possibly issue affects JS / all.
- Create a basic grid with options to preventDefaultOnContextMenu set to true
- https://stackblitz.com/edit/angular-ivy-q8ibdr

**Current behavior**
In 25.2 and 25.3 the browser default context menu **is** displayed.
See: https://stackblitz.com/edit/angular-ivy-q8ibdr

**Expected behavior**
Expected browser default context menu to be prevented as in 25.1.
Working OK in 25.1: https://stackblitz.com/edit/angular-ivy-5ukxva

**AG Grid version:** 
* Affects versions: 25.2, 25.3
* Seems to be OK in latest, but asking maybe a fix could be found for the 25 versions?

**Browser:**
Tried in Chrome.

**Language:** [all | TypeScript X.X | ES6/7 | ES5]
See StackBlitz, used Angular and Typescript.

**Comment by makinggoodsoftware** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by kiril-matev** [comment]
Hello!

Thank you for reporting this - much appreciated!

This issue was logged with the following reference and description:
**AG-5735 [Regression] When setting preventDefaultOnContextMenu=true and right-clicking the ag-grid-community grid, the default context menu is still displayed**

As you can see in our changelog page, this issue is resolved in AG Grid v26.2
https://www.ag-grid.com/changelog/?fixInVersion=26.2.0

We do not release patches for old versions, but instead roll all fixes in future versions. Please upgrade to AG Grid v26.2 or later to resolve this issue.

Thanks for bringing this up - we appreciate your time.

**Comment by makinggoodsoftware** [comment]
Hi,

In order to help us keep our issues queue clear, we are going to close this issue

The official way to check the status for an issue raised here is to check its AG-xxxx ID against our pipeline and changelog

…the pipeline (if not yet developed)

https://www.ag-grid.com/ag-grid-pipeline/

... or the changelog, (if the issue has been resolved)

https://www.ag-grid.com/ag-grid-changelog/

We keep issues raised in the pipeline through github open for a while so that the community can add some feedback to them, and close them when there is no activity for a while.

If you want to add further feedback to a closed ticket with this comment, we would recommend opening a new ticket as we don’t monitor closed issues.
